src/containerapp/azext_containerapp/_params.py

```diff
@@ -144,14 +151,11 @@ def load_arguments(self, _):
     with self.argument_context('containerapp ingress traffic') as c:
         c.argument('traffic_weights', nargs='*', options_list=['--traffic-weight'], help="A list of revision weight(s) for the container app. Space-separated values in 'revision_name=weight' format. For latest revision, use 'latest=weight'")
 
-    with self.argument_context('containerapp secret set') as c:
+    with self.argument_context('containerapp secret') as c:
```

Comment:
Should this be 'list' ? i.e containerapp secret list?

Reply:
Generalized it for `containerapp secret` so that all secret commands with these parameters will have the helptext